lightning-invoice/src/utils.rs

```diff
@@ -811,6 +812,7 @@ mod test {
 		assert_eq!(invoice.min_final_cltv_expiry(), MIN_FINAL_CLTV_EXPIRY as u64);
 		assert_eq!(invoice.description(), InvoiceDescription::Direct(&Description("test".to_string())));
 		assert_eq!(invoice.route_hints().len(), 2);
+		assert_eq!(invoice.expiry_time(), Duration::from_secs(3600));
```

Comment:
We may want to use something other than `DEFAULT_EXPIRY_TIME` in tests since `Invoice::expiry_time` will return that if not set (i.e., just updating the test and not the implementation here would have caused this assertion to pass).

Reply:
Just have a power cut in 2 min for the next two hours so might be after that only.